src/NServiceBus.RabbitMQ.AcceptanceTests/NServiceBus.RabbitMQ.AcceptanceTests.csproj

```diff
@@ -21,6 +21,7 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
```

Comment:
Most of our other repos that I've worked in have this flag set to true, (and when I was creating new projects, @SimonCropp reviewed them and asked for this flag to be true as well) so I assumed it was a standard for us. I think it makes sense to have this as a default anyway. :+1: to treating errors as warnings here.